complaint-urls.json

```diff
@@ -0,0 +1,21 @@
+{
```

Comment:
Just a quick note, I created a new file here rather than appending onto the `videos.json` file since it will allow for only having to maintain the complaint urls in one location, since there many locations with multiple videos.